examples/appmesh-mtls/versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.72"
+    }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.14"
+    }
+  }
+
+  backend "local" {
```

Comment:
please follow the existing format - see https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/examples/tls-with-aws-pca-issuer/versions.tf#L19-L24

```hcl
  # ##  Used for end-to-end testing on project; update to suit your needs
  # backend "s3" {
  #   bucket = "terraform-ssp-github-actions-state"
  #   region = "us-west-2"
  #   key    = "e2e/appmesh-mtls/terraform.tfstate"
  # }
```